Form/Type/ImageType.php

```diff
@@ -195,4 +196,4 @@ private function addAspectRatio(array &$aspectRatios, $key, $value, $checked = f
 
         $aspectRatios[$key] = new AspectRatio($value, $label, $checked);
     }
-}
+}
```

Comment:
Could you please add a line at end of file to be PSR2 compliant so that CI pass?

Once it is done I'll merge your PR.